Definitions.md

```diff
@@ -0,0 +1,16 @@
+#Definition Of Terms
+
+##Job
```

Comment:
@denis-yuen I like that. Regarding `tool` my thought there was that regardless of what we want people use that word to mean "something I run" but the granularity of "something" can be wildly different. My inclusion of it here was more of a nod to it being a fairly generic word that in and of itself doesn't carry any concrete information. Perhaps I could say something to that effect in the block?